src/main/java/org/folio/rest/impl/UploadBatchVoucherExportHelper.java

```diff
@@ -149,8 +158,8 @@ private Future<Void> updateHolderWithBatchVoucher(BatchVoucherUploadHolder uploa
 
   private Future<Void> failUploadUpdate(BatchVoucherExport bvExport, Throwable t) {
     if (bvExport != null) {
-      if (!CREDENTIALS_NOT_FOUND.equals(t.getMessage())
-                      && !(t instanceof URISyntaxException)) {
+      if (Boolean.FALSE.equals(URL_NOT_FOUND_FOR_FTP.equals(t.getMessage()) || URI_SYNTAX_ERROR.equals(t.getMessage())
+        || CREDENTIALS_NOT_FOUND.equals(t.getMessage()))) {
```

Comment:
After talk with @Abdulkhakimov, decied that the cases connected with not found exception shouildn't change the status.
However, we can discuss this part